### PR TITLE
#22 Get kind from local path

### LINKED ISSUE #22: Check if a file or directory
There should be a method to check if a path represents an existing file or a directory.

Perhaps something like this?

```csharp
struct LocalPath
{
  // …
  // null -> not exists
  public FileEntryKind? ReadKind() { … }
}

enum FileEntryKind
{
  File,
  Directory,
  // others: Junction, Symlink?
}
```

### PR DESCRIPTION
**Pull request by Kataane**
Add method that returns type from local path

Closes #22.

### CODE REVIEW

File: TruePath/LocalPath.cs [hunk 1]
```diff
@@ -114,4 +116,83 @@ public bool IsPrefixOf(LocalPath other)
...
+    /// <summary>
+    /// Determines the type of the file system entry (file, directory, symlink, or junction) for the given path.
+    /// </summary>
+    /// <returns>
+    /// A <see cref="FileEntryKind"/> enumeration value representing the type of the file system entry, or <see langword="null"/> if the entry does not exist.
+    /// </returns>
+    /// <remarks>
+    /// This method checks if the specified path represents a file, directory, symbolic link, or junction. On Windows, it uses <see cref="DiskUtils.IsJunction"/> to identify junctions and checks for the <see cref="FileAttributes.ReparsePoint"/> flag to identify symbolic links.
+    /// </remarks>
+    public FileEntryKind? ReadKind()
```
Comment (ForNeVeR) [inline TruePath/LocalPath.cs:129]: I'd prefer this to be ported to `AbsolutePath` instead.

**Review by ForNeVeR — CHANGES_REQUESTED** [review]
Great! There's couple of small issues.

**Comment by Kataane** [comment]
@ForNeVeR fixed

**Review by ForNeVeR — APPROVED** [review]
Thanks!

**Pull request merged**